assemblies/karaf/features/src/main/resources/features.xml

```diff
@@ -20,11 +20,11 @@
     <bundle>mvn:pentaho/pentaho-data-profiling-model-core/${project.version}</bundle>
     <bundle>mvn:pentaho/pentaho-data-profiling-model-metrics/${project.version}</bundle>
     <bundle>mvn:pentaho/pentaho-data-profiling-model-metric-contributors-percentile/${project.version}</bundle>
-    <bundle>mvn:pentaho/pentaho-notification-webservice-bundle/${project.version}</bundle>
+    <bundle>mvn:pentaho/pentaho-notification-webservice-bundle/${notification.version}</bundle>
```

Comment:
@JoaoLMPereira currently the build team is working on replacing properties besides project.version but these changes will break the nightly until they get it working